lib/modules/manager/gradle/extract.ts

```diff
@@ -70,6 +80,15 @@ export async function extractAllPackageFiles(
       } else if (isTOMLFile(packageFile)) {
         const updatesFromCatalog = parseCatalog(packageFile, content);
         extractedDeps.push(...updatesFromCatalog);
+      } else if (isGcvPropsFile(packageFile)) {
```

Comment:
There is an || below, meaning this will be true if *either* `versions.props` or `versions.lock` exists but there's no guarantee or check that both exists.

`fileContents[VERSIONS_PROPS]!` or `fileContents[VERSIONS_LOCK]!` could be null pointers then...